pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -19,35 +19,15 @@
 
 stdenv.mkDerivation rec {
   pname = "util-linux" + lib.optionalString (!nlsSupport && !ncursesSupport && !systemdSupport) "-minimal";
-  version = "2.39";
+  version = "2.39.1";
 
   src = fetchurl {
     url = "mirror://kernel/linux/utils/util-linux/v${lib.versions.majorMinor version}/util-linux-${version}.tar.xz";
-    hash = "sha256-MrMKM2zakDGC7WH+s+m5CLdipeZv4U5D77iNNxYgdcs=";
+    hash = "sha256-iQro/4ECR70Z4nTfdug3HSAs2gGtJ3aBsOqI7qoAKGs=";
   };
 
   patches = [
     ./rtcwake-search-PATH-for-shutdown.patch
-
```

Comment:
looks like these changes make `fetchpatch` an unused argument. feel free to remove it, if you want to (or don't, just wanted to point it out).